helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/ResourceAssignmentOptimizerAccessor.java

```diff
@@ -104,6 +105,8 @@ private static class InputJsonContent {
   }
 
   private static class InstanceChangeMap {
+    @JsonProperty("InstanceConfigs")
+    JsonNode instanceConfigs;
```

Comment:
As this is a new field, so will there be a case where client might pass json with out this property?

Should we use [JsonIgnoreProperties.html#ignoreUnknown](http://fasterxml.github.io/jackson-annotations/javadoc/2.9/com/fasterxml/jackson/annotation/JsonIgnoreProperties.html#ignoreUnknown() )

Reply:
Yes, this case is covered in the unit tests that already existed. JsonProperty does not require that this field be there in InstanceChangeMap.